migrations/15.0.1.4.0/post-migration.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from psycopg2.extensions import AsIs
+
+from odoo import SUPERUSER_ID, api
+
+
+def migrate(cr, version):
+    """Put partner_id in partner_ids"""
+    env = api.Environment(cr, SUPERUSER_ID, {})
```

Comment:
Use `openupgradelib.m2o_to_x2x` method instead.

Reply:
could you please check it now? Thanks!